wsl-vpnkit

```diff
@@ -45,6 +45,7 @@ command -v iptables-legacy >/dev/null && alias iptables=iptables-legacy
 
 run () {
     echo "starting vm and gvproxy..."
+    printenv
```

Comment:
I guess this was just for debugging purposes?

Reply:
Yeah, made a reflection for myself that it would be usefull to have it.
But, I removed it if you find it not suitable to stay in "production" code